File: app/Domain/DTO/PlayerDTO.cs
namespace Domain.DTO
{
	public class PlayerDTO
	{
		public int Id { get; set; }
		public string? Name { get; set; }
		public string? Nickname { get; set; }
		public int Age { get; set; }
		public string? Nationality { get; set; }
		public string? IsActive { get; set; }
		public int TeamId { get; set; }
		public List<TrophyDTO>? Trophies { get; set; }
	}
}

File: client/src/models/DTO/PlayerDTO.ts
import { Trophy } from '../Trophy';
import { TrophyId } from '../TrophyId';
import { TrophyObj } from '../TrophyObj';

export interface PlayerDTO {
    id?: number | string;
    nickname: string;
    name: string;
    age: number | string;
    nationality: string;
    isActive: boolean | string;
    teamId?: number | string;
    trophies?: Trophy[] | TrophyObj[] | TrophyId[];
}
